packages/theme/css/characteristics/text.css

```diff
@@ -79,6 +80,10 @@
   --uitk-text-code-fontFamily: var(--uitk-typography-fontFamily-code);
 }
 
+.backwardsCompat {
+  --uitk-text-lineHeight-backwardsCompat: 1.3;
```

Comment:
this is the same as text-lineheight above?

Reply:
text-lineheight is the default which may be overridden, but --uitk-text-lineHeight-backwardsCompat will take precedence when .backwardsCompat class is present

Reply:
i understand, backwards compat should be added to components though rather than characteristics (doesn't really make sense for characteristics to have backwards compatibility) - also we tend to set attributes directly in the backwardsCompat class (search through current usage to see, e.g. RadioIcon.css) not have a special token for it, since one day backwards compat will probably be deleted and it's best to not encourage having these tokens floating around. it is better to use the CSS API in backwards compat class (e.g. PillCheckbox.css), if not set attribute directly as mentioned

Reply:
see https://github.com/jpmorganchase/uitk/pull/260